python/samples/concepts/prompt_templates/configuring_prompts.py

```diff
@@ -8,61 +8,76 @@
 from semantic_kernel.functions import KernelArguments
 from semantic_kernel.prompt_template import InputVariable, PromptTemplateConfig
 
+kernel = Kernel()
 
-async def main():
-    kernel = Kernel()
+useAzureOpenAI = False
+model = "gpt-35-turbo" if useAzureOpenAI else "gpt-3.5-turbo"
+service_id = model
```

Comment:
Do we need `service_id` if the value is `model` unmodified?

Reply:
Not really. The service_id is there just to be explicit about it as it appeared to trip some people up.